Sources/Extensions/SwiftStdlib/StringExtensions.swift

```diff
@@ -105,7 +105,17 @@ public extension String {
 		return String(first)
 	}
 
-	#if canImport(Foundation)
+    /// SwifterSwift: First character of string uppercased(if applicable) while keeping the original string.
+    ///
+    ///        "hello world".firstCharacterUppercased -> "Hello world"
+    ///        "".firstCharacterUppercased -> ""
+    ///
+    public var firstCharacterUppercased: String {
```

Comment:
Actually, I think we should make this a method and not a property. Most `String` operations in Swift are methods because they're not constant time operations

Reply:
Let's follow the same pattern as `uppercased` @happiehappie

Reply:
Agreed.
1) A copy of a string with its first letter uppercased isn't a property of the string, it's a transformation, so semantically it follows that it's a function.
2) I believe we have an agreed standard that anything that isn't constant time is a function, not a property.